modules/core/src/views/view-manager.js

```diff
@@ -212,6 +267,11 @@ export default class ViewManager {
     });
   }
 
+  _getViewState(viewId) {
+    // Backward compatibility: view state for single view
+    return this.viewState[viewId] || this.viewState
```

Comment:
Whoa, you are actually adding support for multiple viewStates?

I was thinking about doing it this way... Doesn't it also feel a little strange to you to have an object of objects? No way to structurally discern what level of nesting the app is providing...?

Reply:
A little... Let's have an API audit next week.